src/github.rs

```diff
@@ -41,6 +42,24 @@ impl Client {
             .collect())
     }
 
+    // https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#list-organization-repositories
+    pub async fn organisation_repos(
```

Comment:
Yeah I started with this library but it didn't have the two main features I was after at the time from what I could tell. I started adding them but in the end I removed it as it was a lot quicker to write a small amount of HTTP code.

Reply:
Luckily GitHub's API is very easy to work directly with.

Reply:
> Yeah I started with this library but it didn't have the two main features I was after at the time from what I could tell. I started adding them but in the end I removed it as it was a lot quicker to write a small amount of HTTP code.

Octocrab comes with a [HTTP API](https://github.com/XAMPPRocky/octocrab#http-api) for exactly this use case.

Reply:
Oh cool! I didn't know about that.

Looks like pretty much equivalent code as is in there now, so I'm happy with the reqwest approach. I already know that API and the GitHub API is straightforward so I was able to knock it together in less time than I spent reading the library.

Reply:
> Looks like pretty much equivalent code as is in there now, so I'm happy with the reqwest approach. I already know the reqwest API, the GitHub API is straightforward, and these are such simple calls, so I was able to knock it together in less time than I spent reading the library.

My concern is less to do with right now, but much more about maintainability in the future. In my experience, these bots and integrations tend not receive much if any attention until a new feature is needed, which can often be a significant amount of time apart, and being written by completely different people. So while it was easier for you to write right now, I think it will be harder for other people to continue your work and maintain it later, as future additions that use other parts of GitHub's API will have to also be written from scratch, given documentation, and tested, as opposed to using a library that's being used by other companies and individuals who are motivated to provide fixes and additions. It's technical debt, it's better for us to just maintain the parts of the integration that are specific to Embark's organisation.

Reply:
I'm not sure I would agree with you that it is technical debt, it's a very small amount of code written against an API that has not changes in a long time, and future additions could take any shape.

I would actually normally recommend using fewer libraries for code that is changed less frequently as then there is less maintained to keep up with, and you don't need to take action when those extra deps get deprecated. The fewer external factors the less bit-rot there is. If we were using a higher level language here I would be advocating for only using the standard library for this sort of thing.

Either way I don't think this matters here. I wrote this code in 30 minutes and it could be rewritten from scratch in the same amount of time using whatever tools future Embarkers see fit.